apps/utils/src/device/isAOS/isAOS.ts

```diff
@@ -0,0 +1,4 @@
+import { hasUserAgent, HasUserAgentParams } from '@/hasUserAgent';
+
+export const isAOS = (params?: Pick<HasUserAgentParams, 'userAgent'>) =>
+  hasUserAgent({ target: /Android/i, userAgent: params?.['userAgent'] });
```

Comment:
q: target 은 상수론 관리 안 해도 될까요?

p3
```suggestion
  hasUserAgent({ target: /Android/i, userAgent: params?.userAgent });
```